Assets/_BForBoss/_Core/Scripts/Managers/WeaponRaceCourseWorldManager.cs

```diff
@@ -0,0 +1,122 @@
+using System.Linq;
+using Perigon.Analytics;
+using Perigon.Entities;
+using Perigon.Leaderboard;
+using Perigon.Utility;
+using Perigon.Weapons;
+using Sirenix.OdinInspector;
+using Sirenix.Utilities;
+using UnityEngine;
+using UnityEngine.Rendering;
+
+namespace BForBoss
+{
+    public class WeaponRaceCourseWorldManager: BaseWorldManager
+    {
+        [Title("Component")] 
+        [SerializeField] private TimeManager _timeManager = null;
+        [SerializeField] private CheckpointManager _checkpointManager = null;
+        [Title("Weapon/Equipment Component")] 
+        [SerializeField] private WeaponsManager _weaponsManager = null;
+        [SerializeField] private EquipmentBehaviour _equipmentBehaviour = null;
+        [SerializeField] private AmmunitionCountViewBehaviour _ammunitionCountView = null;
+        [SerializeField] private ReloadViewBehaviour _reloadView = null;
+        
+        [Title("User Interface")]
+        [SerializeField] private TimerViewBehaviour _timerView = null;
+        [SerializeField] private ForcedSetUsernameViewBehaviour _forcedUploadView = null;
+
+        [Title("Effects")] 
+        [SerializeField] private Volume _deathVolume = null;
+
+        private readonly PerigonAnalytics _perigonAnalytics = PerigonAnalytics.Instance;
+        private PostProcessingVolumeWeightTool _postProcessingVolumeWeightTool = null;
+        private DetectInput _detectInput = new DetectInput(); //Placeholder, remove this after finishing the timed leader board stuff
```

Comment:
Does this comment still stand? should we remove this variable?

Reply:
uhhh i wanna keep it until we remove the whole leaderboard